handlers.go

```diff
@@ -557,7 +522,7 @@ func mkSelectString(params map[string]string) (string, []interface{}, error) {
 }
```

Comment:
In fact apid-core has implemented its DB service,  and you can make use of it if you want.
If you want your own DB service, you don't have to assemble the query string by your self. You can use sql.Stmt
You may check the code of `getDeployments` func in `apidGatewayDeploy`. 
https://github.com/30x/apidGatewayDeploy/blob/master/data.go#L162